sites/frontend/components/Header/Nav/OpenMainMenu/index.js

```diff
@@ -0,0 +1,171 @@
+// @flow
+import { styled, Component } from '@guardian/guui';
+
+import { desktop } from '@guardian/pasteup/breakpoints';
+import { screenReaderOnly } from '@guardian/pasteup/mixins';
+import { headline } from '@guardian/pasteup/fonts';
+
+import VeggieBurger from './VeggieBurger';
+
+const ScreenReadable = styled('span')(screenReaderOnly);
+
+const navPrimaryColour = '#121212';
+const navSecondaryColour = '#5d5f5f';
```

Comment:
Maybe these can move to `pasteup`? We'll probably use them in the slim nav, and maybe other places

Reply:
Agreed, but I'd like to raise a separate PR where I tidy up the colour codes